README.md

```diff
@@ -9,6 +9,8 @@ This app is one of several App Templates included in Foundry that you can use to
 preconfigured capabilities aligned to its business purpose. Deploy this app from the Templates page with a single click in the Foundry UI, or 
 create an app from this template using the CLI.
 
+Please complete the Foundry Quickstart before deploying this sample. You can find it in Falcon under Foundry > Learn.
```

Comment:
Suggestion: We can use all clouds links for this, as it's done in **Deploy an app** below.

Reply:
What I would do is just  Foundry > Learn (US, US-2, EU) with the links in the respective cloud name.

Reply:
@mraible the problem with removing the `dist` directory is that someone who clones this repo now needs to install node, etc and run a build for the UI page before deploying the app

Reply:
Doesn't the foundry CLI provision those binaries itself ?

Reply:
@rhinchey-cs Good point. I removed the ignore.

Reply:
I rebuilt the `dist` directory with `yarn build` and you can see all the filenames changed. I believe that's what triggered me to ignore the directory. Also, I have the following files after running this command. Should these be ignored?

```
.yarn/
.yarnrc.yml
```

Reply:
> Doesn't the foundry CLI provision those binaries itself? Otherwise how it is run ?

Does using `foundry ui run` require node installed? It might make sense to add it to the dependencies if it's implicitly required for some of the app development lifecycle.

Reply:
it does not require node if the UI page/extension is already built